Exiled.API/Enums/CameraType.cs

```diff
@@ -26,6 +26,9 @@ public enum CameraType
         /// </summary>
         Unknown,
 
+        /// <summary>
+        /// Represents the cameras in Enterance Zone.
+        /// </summary>
```

Comment:
This will be applied to first value in this region (EzChkptHall). You can't make a summary for region

Reply:
Removed